lib/ios-crash-log.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import _fs from 'fs';
+import _ from 'lodash';
+import glob from 'glob';
+import { getLogger } from 'appium-support';
+import B from 'bluebird';
+
+let fs = {
+  stat: B.promisify(_fs.statSync),
+  readFile: B.promisify(_fs.readFile)
+};
+require('date-utils');
+let logger = getLogger('ios-crash-log');
+
+class IosCrashLog { 
+  constructor() {
+    this.prevLogs = [];
+    this.logsSinceLastRequest = [];
+    this.crashDir = path.resolve(process.env.HOME, 'Library', 'Logs', 'DiagnosticReports');
+  }
+
+  async getCrashes() {
+    try {
+      let crashFiles = await glob(`${this.crashDir} /*.crash`);
+      return crashFiles;
+    } catch (err) {
+      logger.error('There was a problem getting the crash list');
+      throw err;
+    }
+  }
+
+  async filesToJSON(arr) {
+    let logs = [];
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
when iterating we prefer to use `let...of` nowadays. I'd change `arr` to `files` and then do:

```js
for (let file of files) {
```